templates/default/src/pages/send-token.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Select,
+  Stack,
+  useClipboard,
+  useToast,
+} from "@chakra-ui/react";
+import { useAccount, useActiveChain, useSendTokens } from "graz";
+import type { FormEvent } from "react";
+import { useMemo, useRef, useState } from "react";
+
+const SendToken = () => {
+  const { data } = useAccount();
+  const activeChain = useActiveChain();
+  const transactionHash = useRef("");
+  const { onCopy } = useClipboard(transactionHash.current);
+  const toast = useToast();
+
+  const { sendTokensAsync, isLoading, isSuccess } = useSendTokens({
+    onError: () => {
```

Comment:
You can pull the argument from here
```
(err, args) => {
  toast({
 ...
description: `Failed send token to ${args.recipientAddress} ` // so you don't need to get it from formData
})
}
```

Reply:
I try to implement this, but I got this typescript error
```
const { sendTokensAsync, isLoading } = useSendTokens({
    onError: (err, args: SendTokensArgs) => {
      return toast({
        status: "error",
        title: "Send token fail",
        description: `Failed send token to ${args.recipientAddress}`,
      });
    },
  });
```

error
<img width="614" alt="image" src="https://user-images.githubusercontent.com/43783037/195719950-1e0dc332-b5c1-42c4-8a2b-fe6d1ec892d4.png">

Reply:
It's a bug in graz, We are not add types into `MutationEventArgs` already create the issue https://github.com/strangelove-ventures/graz/issues/44. for workaround you can do this
```
const { sendTokensAsync, isLoading } = useSendTokens({
    onError: (_, data) => {
     const args = data as SendTokenArgs
      return toast({
        status: "error",
        title: "Send token fail",
        description: `Failed send token to ${args.recipientAddress}`,
      });
    },
  });
```

Reply:
Okay @codingki , I Already push this code